ethkey/src/signature.rs

```diff
@@ -44,8 +45,32 @@ impl Signature {
 		self.0[64]
 	}
 
+	/// Encode the signature into VRS array (V altered to be in "Electrum" notation).
+	pub fn into_vrs(self) -> [u8; 65] {
+		let mut vrs = [0u8; 65];
+		vrs[0] = self.v() + 27;
+		vrs[1..33].copy_from_slice(self.r());
+		vrs[33..65].copy_from_slice(self.s());
+		vrs
+	}
+
+	/// Parse bytes as a signature encoded as VRS (V in "Electrum" notation).
+	/// May return empty (invalid) signature if given data has invalid length.
+	pub fn from_vrs(data: &[u8]) -> Self {
+		if data.len() != 65 || data[0] < 27 {
```

Comment:
I'm not really trying to validate the signature yet (there is a separate function for this, see `Signature#is_valid`), the check is here to prevent panics/overflows.